client-js/app/elements/labrad-registry.ts

```diff
@@ -220,27 +218,24 @@ export class LabradRegistry extends polymer.Base {
     event.preventDefault();
     var data = JSON.parse(event.dataTransfer.getData('text'));
     console.log('dropped', event.dataTransfer);
-    if (event.ctrlKey) {
-      if (JSON.stringify(this.path) != JSON.stringify(data.path)) {
-        //Comparing strings is lame, I know
-        console.log('copying into window', this.path, data.path);
-        var oldFullPath: string[] = data.path,
-            dialog = this.$.dragCopyDialog,
-            copyOriginName = this.$.originName,
-            copyOriginPath = this.$.originPath,
-            copyNameElem = this.$.dragCopyNameInput,
-            copyPathElem = this.$.dragCopyPathInput;
-        copyNameElem.value = data.name;
-        copyOriginName.textContent = data.name;
-        copyOriginPath.textContent = JSON.stringify(data.path);
-        copyPathElem.value = this.pathToString(this.path);
-        this.$.dragCopyDialog.open();
-        window.setTimeout(() => copyNameElem.$.input.focus(), 0);
+    if (JSON.stringify(this.path) != JSON.stringify(data.path)) {  
```

Comment:
When we pop up a normal directory copy dialog we just use the plain directory name, not "Copy of <name>", so I'd say just do the same here too.